doc/writing_modules.rst

```diff
@@ -27,6 +27,34 @@ which if you are used to XDG_CONFIG paths relates to:
 You can also specify the modules location using ``py3status -i <path to custom
 modules directory>`` in your i3 configuration file.
 
+Publish your custom modules on PyPI
+-----------------------------------
+
+.. note::
+    TODO add a remark here in which version this functionality was added
+
+To share custom modules that are not directly part of py3status you can create
+a package and publish it to PyPI.
+
+If you have installed py3status in a virtualenv (maybe because your custom module
+has dependencies that need to be available) you can also create an installable
+package from your module and publish it on PyPI.
+
+py3status can discover custom modules if they are installed in the same host interpreter
+and if an entry_point in the package ``setup.py`` is defined::
+
+    setup(
+        entry_points={"py3status": ["module = path.to.py3status_module"]},
```

Comment:
Would it be possible to provide a concrete example? I'm not sure I even understand what `path.to.py3status_module` should be I'm sorry

Reply:
Concrete eample is always good, we can use pew3wm module as example. In a nutshell it should point to the object that you want access to when iterating over the enty points (in our case the module where the Py3status class lives)

`entry_points={"py3status": ["module = pew3wm.pewpew"]},` - the pewpew module contains the `Py3status` class.

Reply:
ok then I think having `path.to.py3status_module` is misleading

maybe change to `pypi_package_name.module_name`?